tgui/packages/tgui/interfaces/SlotMachine.js

```diff
@@ -4,112 +4,146 @@
     <Window
       title="Slot Machine"
       width={375}
-      height={215}>
+      height={220}
+    >
       <Window.Content>
-        { !scannedCard ? (
-          <InsertCard />
-        ) : (
-          <Box>
-            { busy ? (
-              <BusyWindow />
-            ) : (
-              <SlotWindow />
-            )}
-          </Box>
-        )}
+        {
+          !scannedCard
+            ? <InsertCard />
+            : (busy ? <BusyWindow /> : <SlotWindow />)
+        }
       </Window.Content>
     </Window>
   );
 };
 
-const InsertCard = (props, context) => {
+const InsertCard = (_props, context) => {
   const { act } = useBackend(context);
   return (
-    <Box>
+    <>
       <NoticeBox danger>
         You must insert your ID to continue!
       </NoticeBox>
       <Button
         icon="id-card"
-        content={'Insert ID'}
-        onClick={() => act('insert_card')} />
-    </Box>
+        onClick={() => act('insert_card')}
+      >
+        Insert ID
+      </Button>
+    </>
   );
 };
 
-const SlotWindow = (props, context) => {
+const SlotWindow = (_props, context) => {
   const { act, data } = useBackend(context);
-  const { scannedCard, money, account_funds, plays, wager } = data;
+  const {
+    account_funds,
+    money,
+    plays,
+    scannedCard,
+    wager,
+  } = data;
 
   return (
-    <Box>
+    <>
       <NoticeBox success>
         <marquee> Wager some credits! </marquee>
       </NoticeBox>
-      <Box mb="0.5em">
-        <strong>Your card: </strong>
-        <Button
-          icon="eject"
-          content={scannedCard}
-          tooltip="Pull Funds and Eject Card"
-          tooltipPosition="bottom-right"
-          onClick={() => act('eject')} />
-      </Box>
-      <Box>
-        <strong>Account Balance:</strong>
-        <Icon name="dollar-sign" /> { account_funds }
-        {' '}
-        <Button
-          content="Cash In"
-          tooltip="Add Funds"
-          tooltipPosition="right"
-          onClick={() => act('cashin')} />
-        {' '}
-        <Button
-          content="Cash Out"
-          tooltip="Pull Funds"
-          tooltipPosition="right"
-          onClick={() => act('cashout')} />
-      </Box>
-      <Box>
-        Amount Wagered:
-        <NumberInput
-          minValue={20}
-          maxValue={1000}
-          value={wager}
-          format={value => "$" + value}
-          onDrag={(e, value) => act('set_wager', {
-            bet: value,
-          })}
-        />
-      </Box>
-      <Box mb="0.75em">
-        <strong>Credits Remaining:</strong>
-        <Icon name="dollar-sign" /> { money }
-      </Box>
-      <BlockQuote>
-        { plays } attempts have been made today!
-      </BlockQuote>
-      <Divider />
-      <Button
-        icon="dice"
-        content="Play!"
-        tooltip="Pull the lever"
-        tooltipPosition="right"
-        onClick={() => act('play', {
-          bet: wager,
-        })} />
-    </Box>
+      <Stack vertical>
```

Comment:
Definitely some out-of-scope changes in here, did them while trying to work out some of the layout things for the popper replacements. Happy to revert and do in a separate PR if needed; one of the perks of being a dev is that I can do what I want.